automatic-off-session-payment/process.edn

```diff
@@ -0,0 +1,169 @@
+{:format :v3,
+ :transitions
+ [{:name :transition/enquire,
+   :actor :actor.role/customer,
+   :privileged? true,
+   :actions [
+     {:name :action/create-pending-booking}
+     {:name :action/privileged-set-line-items}],
+   :to :state/pending}
+  {:name :transition/accept,
+   :actor :actor.role/provider,
+   :actions [{:name :action/accept-booking}],
+   :from :state/pending,
+   :to :state/accepted}
+  {:name :transition/decline,
+   :actor :actor.role/provider,
+   :actions [{:name :action/decline-booking}],
+   :from :state/pending,
+   :to :state/declined}
+  {:name :transition/expire,
+   :at
+   {:fn/min
+    [{:fn/plus
+      [{:fn/timepoint [:time/first-entered-state :state/pending]}
+       {:fn/period ["P6D"]}]}
+     {:fn/minus
+      [{:fn/timepoint [:time/booking-end]} {:fn/period ["P1D"]}]}]},
+   :actions [{:name :action/decline-booking}],
+   :from :state/pending,
+   :to :state/declined}
+  {:name :transition/mark-pending-payment,
+   :at
+   {:fn/minus
+    [{:fn/timepoint [:time/booking-start]} {:fn/period ["P1D"]}]},
+   :from :state/accepted,
+   :to :state/pending-payment,
+   :actions []}
+  {:name :transition/auto-payment,
+   :from :state/pending-payment,
+   :to :state/paid,
+   :at
+   {:fn/plus
+    [{:fn/timepoint [:time/first-entered-state :state/pending-payment]}
+     {:fn/period ["PT5M"]}]},
+   :actions
+   [{:name :action/stripe-create-payment-intent, :config { :use-customer-default-payment-method? :true }}
+    {:name :action/stripe-capture-payment-intent}]}
+  {:name :transition/initiate-payment,
+   :actor :actor.role/customer,
+   :from :state/pending-payment,
+   :to :state/pending-confirmation,
+   :actions [{ :name :action/stripe-create-payment-intent }]}
+  {:name :transition/confirm-payment,
+   :actor :actor.role/customer,
+   :from :state/pending-confirmation,
+   :to :state/paid,
+   :actions [{ :name :action/stripe-confirm-payment-intent },
+    { :name :action/stripe-capture-payment-intent }]}
```

Comment:
Indentation + wrapping whitespaces inside action maps. The actions array could also be on it's own two lines like the ones with more than one action definition seem to be.

Reply:
Fixed!